src/main/java/com/internet/mixshop/controller/InjectDataController.java

```diff
@@ -26,10 +28,17 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
         User grinch = new User("Mr.Grinch", "login", "1234");
         User morty = new User("Morty", "rick", "jessika");
+        User admin = new User("admin", "admin", "1234");
+        admin.setRoles(Set.of((Role.of("ADMIN"))));
+        grinch.setRoles(Set.of((Role.of("USER"))));
+        morty.setRoles(Set.of((Role.of("USER"))));
+        userService.create(admin);
         userService.create(grinch);
         userService.create(morty);
+        ShoppingCart cartAdmin = new ShoppingCart(admin.getId());
```

Comment:
I think you don't need to create shopping carts for ADMINs because they can't add products to or remove products from the shopping carts, and they can't compete orders.

Reply:
done